zoo_project.py

```diff
@@ -74,24 +77,88 @@ def __repr__(self):
 
 
 class Animal:
-    """Adds an animal to a cage, required attributes are:
-    animal_species
-    animal_name
-    cage
+
+    """Creates an animal
+    name
     """
 
-    def __init__(self, animal_name, animal_species):
-        self.animal_name = animal_name
-        self.animal_species = animal_species
+    def __init__(self, name):
+        self.name = name
+
+    def __repr__(self):
+        return "{}".format(self.name)
```

Comment:
Where you've used __repr__, you probably want __str__.
http://stackoverflow.com/questions/1436703/difference-between-str-and-repr-in-python
Have a look at the difference between __repr__ and __str__ in our custom Money object in growthstreet.common.money.